kibbeh/src/ui/ScheduledTime.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export const ScheduledTime: React.FC = ({ children }) => {
+  return (
+    <div className="text-primary-200 font-bold">
+      <div className="inline-block mr-2 w-2 h-2 rounded-full bg-primary-200"></div>
```

Comment:
Background for the circle should be `bg-primary-300`

Reply:
ah right! my bad